android/engine/src/main/java/org/smartregister/fhircore/engine/rulesengine/RulesExecutor.kt

```diff
@@ -111,16 +110,20 @@ class RulesExecutor @Inject constructor(val rulesFactory: RulesFactory) {
               relatedResourcesMap = relatedResourcesMap
             )
         }
+      Timber.d(
+        "ListRelatedResources.associate executed in ${ System.currentTimeMillis() - start0 } millisecond(s)"
+      )
 
-      val start2 = System.currentTimeMillis()
-
+      val start = System.currentTimeMillis()
```

Comment:
You can use Kotlin's lambda function [measureTimeMillis](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.system/measure-time-millis.html)